pyobjus/pyobjus.pyx

```diff
@@ -70,6 +76,13 @@ cdef pr(void *pointer):
     # convert a void* to a 0x... value
     return '0x%x' % <unsigned long>pointer
 
+cdef instance_str(py_obj):
+    # Return a repr-like string for any object
+    # guaranteed to be unique for the lifetime of the object
+    return '{}.{}@{}'.format(py_obj.__class__.__module__,
+                             py_obj.__class__.__name__,
+                             hex(builtins.id(py_obj)))
```

Comment:
Just asking, why not just id(py_obj) ?
Maybe having a fqdn as you wrote may be easier for later debugging.